nym-wallet/src/pages/bonding/Bonding.tsx

```diff
@@ -134,6 +147,17 @@ const Bonding = () => {
         />
       )}
 
+      {showModal === 'bond-more' && bondedNode && isMixnode(bondedNode) && (
```

Comment:
do we need these extra checks `bondedNode` && `isMixnode(bondedNode)` because I think the `bond-more` action will only be available on a bonded mixnode.

Reply:
I see, for gateways is only possible to unbond, I didn't know it!

Reply:
@fmtabbara I had to put the condition `isMixnode(bondedNode)` back, because I'm sending the `bondedNode` object to the `DelegateMore` modal.